### PR TITLE
Default configs documentation

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
### Purpose

Documenting default configs
### Issues

#947

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: distribution/resources/conf/micro-gw-default.conf [hunk 1]
```diff
@@ -1,30 +1,55 @@
...
-  trustStorePassword="ballerina"
-  tokenListenerPort=9096
-
+  # Microgateway exposed IP / Host
+  host = "0.0.0.0"
+  # HTTP port that is used to make APIs available to the outside.
+  httpPort = 9090
+  # HTTPS port that is used to make APIs available to the outside and for endpoints(/token, /authorize, /revoke/, userinfo) of Key Manager.
+  httpsPort = 9095
+  # HTTP port for endpoints(/token, /authorize, /revoke/, userinfo) of Key Manager
```
Comment (Rajith90) [inline distribution/resources/conf/micro-gw-default.conf]: HTTPs port

File: distribution/resources/conf/micro-gw-default.conf [hunk 2]
```diff
@@ -1,30 +1,55 @@
...
+  keyStorePassword = "ballerina"
+  # Truststore
+  trustStorePath = "${ballerina.home}/bre/security/ballerinaTruststore.p12"
+  trustStorePassword = "ballerina"
+
+# API Authorization security for backend
 [authConfig]
-  authorizationHeader="Authorization"
-  removeAuthHeaderFromOutMessage=true
+  # Authorization header
```
Comment (Rajith90) [inline distribution/resources/conf/micro-gw-default.conf]: Global Authorization header expected by gateway. Can be override at API level

File: distribution/resources/conf/micro-gw-default.conf (hunk 1 above)
Comment (AmaliMatharaarachchi) [reply distribution/resources/conf/micro-gw-default.conf]: Thanks. Fixed in https://github.com/wso2/product-microgateway/pull/953/commits/3c15c88053510495621ea037d2c045a7c6174314

File: distribution/resources/conf/micro-gw-default.conf (hunk 2 above)
Comment (AmaliMatharaarachchi) [reply distribution/resources/conf/micro-gw-default.conf]: Thanks. Fixed in 3c15c88

**Comment by praminda** [comment]
Travis build is failing due to a known intermittent test failure. Not related to an issue in the PR

**Pull request merged**